webapp/channels/src/components/setting_picture.tsx

```diff
@@ -231,16 +231,10 @@ export default class SettingPicture extends Component<Props, State> {
                     >
                         {imageElement}
                     </div>
-                    <OverlayTrigger
-                        delayShow={Constants.OVERLAY_TIME_DELAY}
+                    <WithTooltip
+                        id='removeIcon'
+                        title={title ?? ''}
```

Comment:
is `title` not always defined? If its not defined we should disable them.

Reply:
There is disabled prop we can utilize

Reply:
> is `title` not always defined? If its not defined we should disable them.

yes, `title` is not always defined, the default value is undefined. When I disable the component when title is undefined, is it still alright to leave it as `title={title ?? ''}` as passing `title={title}` throws a typescript error

Reply:
When an empty string is passed as title? How does the tooltip looks like

Reply:
@Camillarhi can you check the above comment?

Reply:
![image](https://github.com/user-attachments/assets/26c0f44a-f575-4c6d-919b-2890a5a29330)

It looks like this when an empty string is passed a title. I decided to utilize the disabled prop when the title is undefined